daml-lf/spec/daml-lf-1.rst

```diff
@@ -1921,19 +1921,19 @@ need to be evaluated further. ::
    ——————————————————————————————————————————————————— ValScenario
      ⊢ᵥ  s
 
-   ——————————————————————————————————————————————————— ValUnBondedMathContext
+   ——————————————————————————————————————————————————— ValUnboundedMathContext
      ⊢ᵥ  LitRoundingMode
 
 
                            ┌────────┐
    Update Values           │ ⊢ᵥᵤ  u │
                            └────────┘
 
-     ⊢ᵥ  e
+     ⊢ᵥᵤ  e
```

Comment:
It think this one is correct.
```suggestion
     ⊢ᵥ  e
```

Reply:
With your change, `'pure' @τ ()` is not anymore a (update) value.

Reply:
By the way, we need to add an extra `@τ` in `UpdPure` and `ScnPure` typing rules.

Reply:
This one should be `⊢ᵥ  e` since any value can appear in a `pure`.

Reply:
You're right. :man_facepalming: